scheduler/src/test/java/org/cloudfoundry/autoscaler/scheduler/rest/ScheduleRestController_SpecificSchedule_Test.java

```diff
@@ -105,6 +115,50 @@ public void testCreateAndGetSchedules() throws Exception {
 
 		assertCreateAndGetSchedules(multipleAppIds, 1);
 		assertCreateAndGetSchedules(multipleAppIds, 5);
+
+	}
+
+	@Test
+	@Transactional
+	public void testCreateSchedulesMultipleConcurrent() throws Exception {
+		ExecutorService service = Executors.newFixedThreadPool(20);
```

Comment:
The test will fail if the @RequestScope is removed from ValidationErrorResult class.We started on this test, to have concurrent requests for verifying  the ValidationErrorResult scope (which has request scope). Now that we have the initEmpty removed, the test testCreateSchedule_multiple_error is enough.  Should we remove testCreateSchedulesMultipleConcurrent?